include/deal.II/sundials/ida.h

```diff
@@ -274,6 +274,31 @@ namespace SUNDIALS
    * variables is always in instantaneous equilibrium with another set of
    * variables that evolves on a slower time scale.
    *
+   * Another case where we *could* eliminate a variable but do not want to
+   * is where that additional variable is introduced in the first place to work
+   * around some other problem. As an example, consider the time dependent
+   * version of the biharmonic problem we consider in step-47 (as well as some
+   * later ones). The equations we would then be interested in would read
+   * @f{align*}{
+   *   \frac{\partial u(\mathbf x,t)}{\partial t} + \Delta^2 u(\mathbf x,t) &=
+   *f(\mathbf x,t).
```

Comment:
```suggestion
   * f(\mathbf x,t).
```
This is rendered as `*f` in the generated doxygen - I think adding a space will fix it.